nautobot/ipam/models.py

```diff
@@ -836,29 +835,42 @@ def get_first_available_ip(self):
         return f"{next(available_ips.__iter__())}/{self.prefix_length}"
 
     def get_utilization(self):
```

Comment:
Does this help at all with #2248?

Reply:
- With this utilization calculation in place it takes 94 seconds to load the list view with 1000 prefixes. 
- If I short circuit the utilization calculation and just return `UtilizationData(numerator=numerator_set.size, denominator=denominator)` it takes 47 seconds. 
- With the jinja template from #2248 and short circuiting get_utilization it takes 46 seconds.
- With the jinja template and calculating utilization it takes 88 seconds
- When I switch back to `next` it takes 71 seconds

These times are with DJDT enabled

Reply:
Not entirely sure I follow all that. :-) Sounds like the gist is that the jinja template gives only a negligible performance gain at this point?

Reply:
Bottom line is this change makes the utilization calculation slower and it doesn't appear that #2248 is going to help much.